core/api/src/services/mongoose/schema.types.d.ts

```diff
@@ -69,6 +69,7 @@ interface WalletInvoiceRecord {
   currency: string
   timestamp: Date
   selfGenerated: boolean
+  processingCompleted?: boolean
```

Comment:
why do we need to add it as optional? isnt default value working?

Reply:
Isn't default value only used on write? I have a database migration that will go back and populate this field and then i will make it non optional.